app/src/main/java/com/eveningoutpost/dexdrip/GcmActivity.java

```diff
@@ -688,6 +688,13 @@ static void pushCalibration2(double bgValue, String uuid, long offset) {
             GcmActivity.sendMessage(myIdentity(), "cal2", json);
         }
     }
+    public static void pushLibreBlock(String libreBlock) {
```

Comment:
This should also have a `pratelimit` call to ensure that network traffic cannot increase much more than would be expected. So the rate limit should prevent the same or duplicate data being sent multiple times. I assume that this push should not occur more than once every approx 5 minutes?

Reply:
Yes, moved this to a more visible place.